sw/device/tests/BUILD

```diff
@@ -3367,9 +3445,15 @@ opentitan_test(
 opentitan_test(
     name = "spi_host_winbond_flash_test",
     srcs = ["spi_host_winbond_flash_test.c"],
+    broken = fpga_params(tags = ["broken"]),
     exec_env = dicts.add(
         EARLGREY_TEST_ENVS,
         EARLGREY_SILICON_OWNER_ROM_EXT_ENVS,
+        {
+            # See #23558: needs porting to work on the CW340.
```

Comment:
Isn't this test and the above fixed already?

Reply:
Oh yes, I think we did fix this one, I'll re-enable in the next push